libraries/backend/helpers.rb

```diff
@@ -92,6 +92,7 @@ class AzureEnvironments
       active_directory_endpoint_url: MicrosoftRestAzure::AzureEnvironments::AzureCloud.active_directory_endpoint_url,
       storage_endpoint_suffix: MicrosoftRestAzure::AzureEnvironments::AzureCloud.storage_endpoint_suffix,
       key_vault_dns_suffix: MicrosoftRestAzure::AzureEnvironments::AzureCloud.key_vault_dns_suffix,
+      datalake_store_filesystem_endpoint_suffix: MicrosoftRestAzure::AzureEnvironments::AzureCloud.datalake_store_filesystem_endpoint_suffix,
```

Comment:
all changes in this file are not relevant here again

Reply:
done.